root/documentation/compara.conf

```diff
@@ -249,14 +249,30 @@
       </compara>
       <aligned>
         type=Boolean
-        description=Return the aligned string if true. Otherwise, return the original sequence (no insertions)
+        description=Return the aligned string if true. Otherwise, return the original sequence (no insertions). Note: If aligned=1, this will override sequence=none
         default=1
       </aligned>
       <sequence>
         type=Enum(none, cdna, protein)
-        description=The type of sequence to bring back. Setting it to none results in no sequence being returned
+        description=The type of sequence to bring back. Setting it to none results in no sequence being returned. Note: dependant on the setting for "aligned", If aligned=1, this will override sequence=none
         default=protein
       </sequence>
+      <db_type>
+        type=String
+        description=Restrict the search to a database other than the default. Useful if you need to use a DB other than core
+        example=core
```

Comment:
The description says "Restrict the search to a database other than the default. Useful if you need to use a DB other than core" and the example given is core (example=core).
Guess it would be more apt if we give example other than 'core'?!